webservice/src/main/java/dk/dbc/laesekompas/suggester/webservice/solr_entity/SearchEntity.java

```diff
@@ -142,4 +153,50 @@ public int getOrder() {
     public void setOrder(int order) {
         this.order = order;
     }
+
+    public ArrayList<String> getBibIdsInWork() {
+        return bibIdsInWork;
+    }
+
+    public void setBibIdsInWork(ArrayList<String> bibIdsInWork) {
+        this.bibIdsInWork = bibIdsInWork;
+    }
+
+    public static ArrayList<SearchEntity> searchResultsIntoSearchEntities(SolrDocumentList docs, Integer order) {
+        ArrayList<SearchEntity> buffer = new ArrayList<>();
+        // Converting search results to SearchEntity
+        for (SolrDocument doc : docs) {
```

Comment:
I'd consider rewriting this to a function on single SolrDocument objects. (and hey - I am taking over this stuff! ;-) )